paper.bib

```diff
@@ -0,0 +1,300 @@
+	date-modified = {2024-10-14 14:36:11 -0700},
+	pages = {133--152},
+	publisher = {Springer},
+	title = {Mesh Curving Techniques for High Order Discontinuous Galerkin Simulations},
+	year = {2015}}
+
+@inproceedings{goutal1999malpasset,
+	author = {Goutal, Nicole},
+	booktitle = {Proceedings of the 4th CADAM meeting, Zaragoza, Spain},
+	date-added = {2024-10-14 13:59:42 -0700},
+	date-modified = {2024-10-14 13:59:42 -0700},
+	pages = {18--19},
+	title = {The {M}alpasset dam failure. {A}n overview and test case definition},
+	year = {1999}}
+
+@article{hervouet1999malpasset,
+	author = {Hervouet, Jean-Michel and Petitjean, Alain},
+	date-added = {2024-10-14 13:59:15 -0700},
+	date-modified = {2024-10-14 13:59:15 -0700},
+	journal = {Journal of hydraulic research},
+	number = {6},
+	pages = {777--788},
+	publisher = {Taylor \& Francis},
+	title = {Malpasset dam-break revisited with two-dimensional computations},
+	volume = {37},
+	year = {1999}}
+
+@article{Bommes:2013lc,
+	abstract = {Triangle meshes have been nearly ubiquitous in computer graphics, and a large body of data structures and geometry processing algorithms based on them has been developed in the literature. At the same time, quadrilateral meshes, especially semi-regular ones, have advantages for many applications, and significant progress was made in quadrilateral mesh generation and processing during the last several years. In this survey we discuss the advantages and problems of techniques operating on quadrilateral meshes, including surface analysis and mesh quality, simplification, adaptive refinement, alignment with features, parametrisation and remeshing.},
+	address = {Chichester, GBR},
+	author = {Bommes, David and L\'{e}vy, Bruno and Pietroni, Nico and Puppo, Enrico and Silva, Claudio and Tarini, Marco and Zorin, Denis},
+	date-added = {2024-10-10 10:26:27 -0700},
+	date-modified = {2024-10-14 14:01:17 -0700},
+	doi = {10.1111/cgf.12014},
+	issn = {0167-7055},
+	issue_date = {September 2013},
+	journal = {Comput. Graph. Forum},
+	keywords = {quad mesh processing, quad mesh generation, mesh parametrization, languages, geometry processing, and systems, I.3.5[Computer Graphics]: Computational Geometry and Object Modeling-Geometric algorithms},
+	month = sep,
+	number = {6},
+	numpages = {26},
+	pages = {51--76},
+	publisher = {The Eurographs Association \& John Wiley \& Sons, Ltd.},
+	title = {Quad-Mesh Generation and Processing: A Survey},
+	url = {https://doi.org/10.1111/cgf.12014},
+	volume = {32},
+	year = {2013},
+	bdsk-url-1 = {https://doi.org/10.1111/cgf.12014}}
+
+@article{tromp2008spectral,
+	author = {Tromp, Jeroen and Komatitsch, Dimitri and Liu, Qinya and others},
+	date-added = {2024-10-10 10:24:09 -0700},
+	date-modified = {2024-10-10 10:24:09 -0700},
+	journal = {Communications in Computational Physics},
+	number = {1},
+	pages = {1--32},
+	title = {Spectral-element and adjoint methods in seismology},
+	volume = {3},
+	year = {2008}}
+
+@webpage{FLUXO,
```

Comment:
@andrewwinters5000 Do you know whether there's a proper way to cite FLUXO?

Reply:
As far as I recall we would cite either the MHD with Marvin or the shock capturing MHD paper with Andrés. I need to double check.

Reply:
OK, thanks!